modules/cnv_control_vm_web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-control-vm-web]]
+== starting, stopping, and restarting a virtual machine
+
+Depending on the state, virtual machines can be xref:cnv_vm_actions_web.adoc#cnv-vm-actions-web[started, stopped, and restarted] using the &#8942; button of each virtual machine in the *Workloads* -> *Virtual Machines* list. 
```

Comment:
Same reason as above. It's moot now, though, since the 'Confirm' button for each of the confirmation pop-up windows is now specific to the actions. I'm going to split this into three modules: Starting a VM, Stopping a VM, and Restarting a VM